Update readme (#212)
PrestaShopCorp/ps_checkout: README.md

PayPal offers a sandbox mode in which an order can be created without
involving actual money.

To enable it, reach the module configuration page, then replace `#...` at the end of the URL with `#/experimental`.

> This route allow you to acces to some experimental features (like paypal sandbox). Don't use it in a production environment until they will be released officialy.

Reviewer: In

> Don't use it in a production environment until they will be released officialy.

What do you mean by `they`?

Author: It should be more clear now? :)

> This route allow you to acces to some experimental features (like paypal sandbox). Don't use it in a production environment until these features will be released officialy.

Reviewer: ```suggestion
This route allow you to acces to some experimental features (like paypal sandbox). Don't use them in a production environment until these features are officially released.
```
